spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -390,6 +405,34 @@ Counters do not have a suffix for the unit.
 application_visitors 80
 ----
 
+==== Concurrent Gauge OpenMetrics Text Format
+
+The values of the concurrent gauge must be the value of `getCount() / getMin() / getMax()`.
+The exposed metric names have suffixes as follows:
+
+|===
+| Value | Suffix
+
+| `getCount` |  `_total`
+| `getMin` |  `_min`
+| `getMax` |  `_max`
+|===
+
+
+The `_total` suffix is not appended if the (translated) original metric name already ends in `_total`.
+
+Concurrent gauges do not have a suffix for the unit.
+
+.Example OpenMetrics text format for a Concurrent Gauge
+[source, ruby]
+----
+# TYPE application_visitors_total counter
+# HELP application_visitors_total The number of parallel invocations of visitorsTotal()
+application_visitors_total 80
+application_visitors_min 20
```

Comment:
The `min` and `max` values should probably have a TYPE line, with the type being... gauge?

Reply:
Yes, this is an error on my side. The original proposal had `_current`

Reply:
Actually I wonder if those should be 3 metrics or 1 metric with 3 different labels
```
method_a_invocations{???="min"}
method_a_invocations{???="max"}
method_a_invocations{???="current"}

```
( I obviously can't come up with a good name for the key here)

Reply:
I would suggest it be 3 different metrics.  When you use labels, the thing you're labeling should make sense to sum or average across all sets of labels -- which wouldn't be the case here.  See https://prometheus.io/docs/practices/naming/ ... 

> As a rule of thumb, either the sum() or the avg() over all dimensions of a given metric should be meaningful (though not necessarily useful). If it is not meaningful, split the data up into multiple metrics. For example, having the capacity of various queues in one metric is good, while mixing the capacity of a queue with the current number of elements in the queue is not.